src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+
+import * as RadixAccordion from '@radix-ui/react-accordion';
+import { IconChevronDown } from '../Icons';
+
+import classNames from 'classnames';
+import styles from './Accordion.module.scss';
+
+const cx = classNames.bind(styles);
+
+export interface AccordionItem {
+  title: string;
+  text: string;
+}
+export interface AccordionProps {
+  items: AccordionItem[];
+  contrast?: 'high' | 'low';
+}
+
+export const Accordion = ({ items, contrast = 'high' }: AccordionProps) => {
+  const [focusState, setFocusState] = useState(false);
+  return (
+    <div className={styles.Accordion} data-contrast={contrast}>
+      <RadixAccordion.Root className={styles.AccordionRoot} type="single" collapsible>
+        {items.map((itm, i) => (
+          <RadixAccordion.Item
+            className={styles.AccordionItem}
+            value={`item-${i}`}
+            key={i}
```

Comment:
Use `itm.title` as the key.

[Index as a key is an anti-pattern](https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/)

Reply:
The title might be the same so I added it like ```key={`${itm.title}-${i}`}```